rasa/shared/core/constants.py

```diff
@@ -1 +1,66 @@
+import rasa.shared.constants as constants
+
 DEFAULT_CATEGORICAL_SLOT_VALUE = "__other__"
+
+USER_INTENT_RESTART = "restart"
+USER_INTENT_BACK = "back"
+USER_INTENT_OUT_OF_SCOPE = "out_of_scope"
+USER_INTENT_SESSION_START = "session_start"
+
+DEFAULT_INTENTS = [
+    USER_INTENT_RESTART,
+    USER_INTENT_BACK,
+    USER_INTENT_OUT_OF_SCOPE,
+    USER_INTENT_SESSION_START,
+    constants.DEFAULT_NLU_FALLBACK_INTENT_NAME,
+]
+
+LOOP_NAME = "name"
+
+ACTION_LISTEN_NAME = "action_listen"
+ACTION_RESTART_NAME = "action_restart"
+ACTION_SESSION_START_NAME = "action_session_start"
+ACTION_DEFAULT_FALLBACK_NAME = "action_default_fallback"
+ACTION_DEACTIVATE_FORM_NAME = "action_deactivate_form"
+ACTION_REVERT_FALLBACK_EVENTS_NAME = "action_revert_fallback_events"
+ACTION_DEFAULT_ASK_AFFIRMATION_NAME = "action_default_ask_affirmation"
+ACTION_DEFAULT_ASK_REPHRASE_NAME = "action_default_ask_rephrase"
+ACTION_BACK_NAME = "action_back"
+ACTION_TWO_STAGE_FALLBACK_NAME = "two_stage_fallback"
+RULE_SNIPPET_ACTION_NAME = "..."
+
+DEFAULT_ACTIONS = [
```

Comment:
I think it would be more consistent to call this `DEFAULT_ACTION_NAMES` since action names all have a `_NAME` suffix
```suggestion
DEFAULT_ACTIONS_NAMES = [
```